site/jekyll/getting-started/tutorial_aspnet4.md

```diff
@@ -1071,14 +1104,14 @@ namespace ReactDemo
 		public static void Configure()
 		{
 			ReactSiteConfiguration.Configuration
-				.AddScript("~/js/remarkable.min.js")
+				.AddScript("~/Scripts/remarkable.min.js")
 				.AddScript("~/Scripts/Tutorial.jsx");
 		}
 	}
 }
 ```
 
-Note that we need a copy of Remarkable in order to load it for server-side rendering. In a production app you'd probably use Bower or npm for this, but for our tutorial you can just [download the file from CDNJS](https://cdnjs.cloudflare.com/ajax/libs/remarkable/1.7.1/remarkable.min.js) and save it into `~/js`.
+Note that we need a copy of Remarkable in order to load it for server-side rendering. In a production app you'd probably use Bower or npm for this, but for our tutorial you can just [download the file from CDNJS](https://cdnjs.cloudflare.com/ajax/libs/remarkable/1.7.1/remarkable.min.js) and save it into `~/Scripts`.
```

Comment:
Leaving remarkable in the `js` folder should prevent it from being unnecessarily compiled through babel.

Reply:
Okay. I can put it back.